build.gradle

```diff
@@ -1,11 +1,17 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    ext.kotlin_version = '1.1.60-eap-43'
     repositories {
         jcenter()
+        maven {
+            url "https://dl.bintray.com/kotlin/kotlin-eap-1.1"
+        }
+        google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.2'
+        classpath 'com.android.tools.build:gradle:3.0.0-beta7'
```

Comment:
Grade tools 3.0.0 is stable no need for suffix

Reply:
good eyes. it's just the exact version i had cached locally, i get 150MB of data for 15 pounds on the airplane 😂 i'm gonna do a big Android Studio, gradle etc version update everywhere once i land and can pull them down!